src/components/atoms/LanguageOption/index.tsx

```diff
@@ -24,13 +24,16 @@ const setLangVer = (props) => {
 const LanguageOption: React.FC<LanguageOptionInterface> = (
     props: LanguageOptionInterface
 ) => {
-    console.log("HEEE: ", props.language)
     return (
         <StyleLanguageOptionContainer>
             {props.language === "en" || props.language === "English" ? 
```

Comment:
What's the purpose of `||`?
Props can be `en` and `English`?
If that's the case, that's odd. 
Props should be either `en` or only `English`?